lucille-core/src/main/java/com/kmwllc/lucille/stage/DictionaryLookup.java

```diff
@@ -42,25 +51,34 @@ public class DictionaryLookup extends Stage {
   private final boolean usePayloads;
   private final UpdateMode updateMode;
   private final boolean ignoreCase;
+  private final boolean setOnly;
+  private final boolean ignoreMissingSource;
 
-  private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+  // Dummy value to indicate that a key is present in the HashMap
+  private static final String[] PRESENT = new String[0];
+  private static final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
```

Comment:
The convention we use is to put "log" in lowercase. That's because it's not a "constant" even though it is static final. In other words, its state can change. You can read a discussion of this question here: https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case

Reply:
makes sense, thank you